lldb/source/Target/ThreadPlanStepThrough.cpp

```diff
@@ -135,12 +137,6 @@ bool ThreadPlanStepThrough::ValidatePlan(Stream *error) {
     return false;
   }
 
-  if (m_backstop_bkpt_id == LLDB_INVALID_BREAK_ID) {
-    if (error)
-      error->PutCString("Could not create backstop breakpoint.");
-    return false;
-  }
```

Comment:
@jimingham likewise, should this be upstreamed? This has less argument in favor, other than keeping diff between llvm/lldb and swift/lldb at a minimum.

Reply:
We should never introduce differences in common code except in the case where we plan to factor something more nicely for incorporating into generic LLDB but haven't figured out how to do it yet.  We are removing a seatbelt here, however.  So it might be nice to have a way to limit when we do that to just step through plans where the failure is not unexpected.  Maybe we should add a way to ask the step-through plan whether this should be a fatal error or not and check that in Validate.

Reply:
Good plan. I realize now the best thing to do am going to first understand why a fallback breakpoint can't be created, and then figure out whether we expect that to always be the case for async stepping, or whether it's a temporary issue that we can and should fix.